### DISCUSSION TITLE
Improve M2A interface display

### DISCUSSION THREAD

**Discussion by lukas-schaetzle** [directus/directus#7870] [Feature Requests]
The current m2a interface display is very limited and looks not so great. Take a look at this example:
![Unbenannt](https://user-images.githubusercontent.com/45203603/132257333-4c0efa92-a2a2-4969-bfd8-144fdb98191a.PNG)

The first item contains a wysiwyg and the second a "multiple asset relation". The miniature images for the asset relation are missing when I click on it, although I've configured the display templates. This seems to be a bug.

Other than that I propose a layout restructure like this:
![Unbenannt2](https://user-images.githubusercontent.com/45203603/132257549-43758964-cfec-4dbe-9d29-3c3158090f11.PNG)

Most notably:
- the assets should be directly visible without clicking on the relation first
- the interface should have an additional option "max-height" which allows to increase the container size, so that more content can be seen or the assets can be seen better

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
